packages/cdk/src/config.ts

```diff
@@ -0,0 +1,214 @@
+import { GetParametersByPathCommand, SSMClient } from '@aws-sdk/client-ssm';
+import {
+  ExternalSecret,
+  ExternalSecretPrimitive,
+  ExternalSecretPrimitiveType,
+  MedplumInfraConfig,
+  MedplumSourceInfraConfig,
+  OperationOutcomeError,
+  badRequest,
+  validationError,
+} from '@medplum/core';
+
+const DEFAULT_AWS_REGION = 'us-east-1';
+const VALID_PRIMITIVE_TYPES = ['string', 'boolean', 'number'];
+
+export async function fetchParameterStoreSecret(path: string, key: string): Promise<string> {
+  const region = DEFAULT_AWS_REGION;
```

Comment:
Region needs to be loaded from the config file for customers who don't use `us-east-1`.